utils/pack.ts

```diff
@@ -21,6 +21,9 @@ async function main() {
     const dir = path.join(rootDir, task.directory);
     const packageJsonPath = path.join(dir, 'package.json');
     const originalPackageObj = await fs.readJson(packageJsonPath);
+    if (originalPackageObj.name === 'api') {
+      continue;
```

Comment:
Lets add a comment here explaining why

I think its because turbo is finding this by parsing the new pnpm-workspace.yml file but we don't want to create a tarball for it since its a Vercel Deployment already?